### ISSUE TITLE
firebase iid 

### ISSUE THREAD

**Issue by paixols** [Iterable/iterable-android-sdk#118]
Kotlin Compiler Error when using Firebase Performance dependency:
>compile group: 'com.google.firebase', name: 'firebase-perf', version: '18.0.1'

```
Supertypes of the following classes cannot be resolved. Please make sure you have the required dependencies in the classpath:
    class com.iterable.iterableapi.IterableFirebaseInstanceIDService, unresolved supertypes: com.google.firebase.iid.FirebaseInstanceIdService	
```

I Tried excluding dependency `firebase-iid` from `firebase-perf` with no success. 
Current iterable version:  `com.iterable:iterableapi:3.1.0`
Gradle plugin version:  `com.android.tools.build:gradle:3.3.2`

**Could this be related to `iterableapi` using an old version of firebase messaging dependency : `com.google.firebase:firebase-messaging:15.0.2`,  which uses `firebase-iid`, and the latest version is: `com.google.firebase:firebase-messaging:19.0.1` ?**

**Comment by vbabenkoru** [comment]
Thanks for the report @paixols!
Is `enableJetifier=true` set in your `gradle.properties` file?

**Comment by vbabenkoru** [comment]
@paixols We've just released a new version, 3.1.2, with support for the new version of `firebase-messaging`, please let me know if that works. Thanks!
